_posts/platform/getting-started/2000-01-01-getting-started-with-metabase.md

```diff
@@ -41,3 +41,13 @@ Last, deploy your application with:
 ```bash
 $ git push scalingo master
 ```
+
+## Update the application
+
+To update your Metabase application to the latest version, you simply need to redeploy the last version of the code source.
```

Comment:
Nothing is "simple". This is one of our advice when writing documentation: https://www.notion.so/scalingooriginal/Writing-Documentation-at-Scalingo-bb7985d6ea16455894417e36ede09cea

```suggestion
To update your Metabase application to the latest version, you need to redeploy the last version of the code source.
```

Reply:
True!

Thanks for your feedback !